src/sentry/sentry_apps/external_requests/alert_rule_action_requester.py

```diff
@@ -25,26 +23,60 @@ class AlertRuleActionResult(TypedDict):
     message: str
 
 
-class AlertRuleActionRequester(Mediator):
-    """
-    Makes a POST request to another service to fetch/update the values for each field in the
-    AlertRuleAction settings schema
-    """
+@dataclass
+class AlertRuleActionRequester:
+    install: SentryAppInstallation | RpcSentryAppInstallation
+    uri: str
+    fields: list[dict[str, str]] = field(default_factory=list)
+    http_method: str | None = "POST"
 
-    install = Param(SentryAppInstallation)
-    uri = Param(str)
-    fields = Param(list, required=False, default=[])
-    http_method = Param(str, required=False, default="POST")
-    using = router.db_for_write(SentryAppInstallation)
+    def run(self) -> AlertRuleActionResult:
+        try:
+            with transaction.atomic(router.db_for_write(SentryAppInstallation)):
+                response = send_and_save_sentry_app_request(
+                    url=self._build_url(),
+                    sentry_app=self.sentry_app,
+                    org_id=self.install.organization_id,
+                    event="alert_rule_action.requested",
+                    headers=self._build_headers(),
+                    method=self.http_method,
+                    data=self.body,
+                )
 
-    def call(self):
-        return self._make_request()
+        except RequestException as e:
+            with isolation_scope() as scope:
+                scope.set_tag("sentry_app", self.sentry_app.slug)
+                scope.set_context(
+                    "alert_rule_action.error",
+                    {
+                        "sentry_app_slug": self.sentry_app.slug,
+                        "install_uuid": self.install.uuid,
+                        "uri": self.uri,
+                        "error_message": str(e),
+                    },
+                )
```

Comment:
In the old implementation this was a logging message. This scope configuration doesn't capture any exceptions or messages so it won't do much. We should keep the logging message though as we don't need to be alerted on requests to sentryapps failing.